components/ws-manager/pkg/manager/internal/grpcpool/pool.go

```diff
@@ -6,133 +6,127 @@ package grpcpool
 
 import (
 	"fmt"
-	"strings"
 	"sync"
-	"time"
 
+	conn_pool "github.com/shimingyah/pool"
```

Comment:
nit: the convention is the camelCase, consider renaming to `connpool` _or_ `connPool`.